bundler.go

```diff
@@ -160,14 +168,16 @@ func absPath(configPath string, defaultPathFn func() (string, error)) (o string,
 func New(c *Configuration) (b *Bundler, err error) {
 	// Init
 	b = &Bundler{
-		appName:           c.AppName,
-		bindPackage:       c.Bind.Package,
-		Client:            &http.Client{},
-		environments:      c.Environments,
-		darwinAgentApp:    c.DarwinAgentApp,
-		resourcesAdapters: c.ResourcesAdapters,
-		ldflags:           c.LDFlags,
-		infoPlist:         c.InfoPlist,
+		appName:            c.AppName,
+		bindPackage:        c.Bind.Package,
+		Client:             &http.Client{},
+		environments:       c.Environments,
+		darwinAgentApp:     c.DarwinAgentApp,
+		resourcesAdapters:  c.ResourcesAdapters,
+		ldflags:            c.LDFlags,
+		infoPlist:          c.InfoPlist,
+		versionElectron:    c.VersionElectron,
```

Comment:
Set both versions to defaults here. Then, at the bottom of this function, do something like this for each version:

```go
if c.VersionElectron != "" {
  b.versionElectron = c.VersionElectron
}
```